_posts/2023-04-26-quarkus-3-0-final-released.adoc

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: 'Quarkus 3.0 released'
+date: 2023-04-26
+tags: release
+synopsis: 'It is our great pleasure to announce our new major release of Quarkus: Quarkus 3.0.'
+author: gsmet
+---
+
+It is our great pleasure to announce the release of Quarkus 3.0,
+our new major release,
+which comes packed with new features and enhancements.
+
+The development of Quarkus 3 started on March 18 last year with the crazy idea
+that we could rewrite the whole tree to migrate to the new `jakarta.*` packages
+while continuing the development of Quarkus 2.
+
+Quarkus 3 is now based on Jakarta EE 10 and, as you can expect,
+it is not the only change, this version comes packed with new features and enhancements
+that we implemented during this period.
+
+This announcement contains a lot of information about the new features and how to upgrade
+so we recommend to read it fully before upgrading to Quarkus 3.
+
+== What's new?
+
+=== Jakarta EE 10
```

Comment:
I wouldn't be so concerned about it being on top of list - its after all the main reason why we went with a 3.x rather than 2.x.

Its also better to compare to Spring 6 announcement which is what actually drove the jakarta change. here its the first thing. https://spring.io/blog/2022/11/16/spring-framework-6-0-goes-ga

Reply:
Like I said, I understand where this is coming from, I would just do it differently myself :)